freeglut/freeglut/src/fg_internal.h

```diff
@@ -572,9 +670,10 @@ do                                            \
 {                                             \
```

Comment:
I don't know why there are two versions of this, but I did try to remove the Windows only version and Visual Studio 2015 failed to compile... so there's something it doesn't like with the second macro.

If you only want the first macro, that would probably work on all platforms (untested). Assuming there isn't someone currently around who wrote the second macro to try and have the compiler inline the function call, without a register being taken up by `func`. If that was the design goal, I'd be amazed if "now", that has any measurable performance difference.